content/docs/next-release/troubleshooting.md

```diff
@@ -4,4 +4,68 @@ description: Solve commonly encountered issues
 weight: 10
 ---
 
-We are preparing a troubleshooting guide for you. In the meantime, make sure to check the blog post ["Help! Something is wrong with my Jaeger installation!"](https://medium.com/jaegertracing/help-something-is-wrong-with-my-jaeger-installation-68874395a7a6)
\ No newline at end of file
+Jaeger is composed of different components, each potentially running in its own host. It might be the case that one of these moving parts isn’t working properly, causing spans to not be processed and stored. When something goes wrong, make sure to check the items listed here.
+
+## Sampling strategy
+
+Before everything else, make sure to confirm what is the sampling strategy. By default, Jaeger uses a probabilistic sampling strategy, with a 1/1000th chance that the span will be reported.
+
+Typically, the sampling strategy can be set via the environment variables `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM`, but refer to the Jaeger Client’s documentation for the language you are using for more details about which sampling strategies are available. When using the Jaeger Java Client, the strategy is usually printed out via the logging facility provided by the instrumented application when creating the tracer:
+
+```
+2018-12-10 16:41:25 INFO  Configuration:236 - Initialized  tracer=JaegerTracer(..., sampler=ConstSampler(decision=true,  tags={sampler.type=const, sampler.param=true}), ...)
+```
+
```

Comment:
Would it be worth clearing stating that, to diagnose why traces may not be received, it is recommended to configure the `const` sampler type with param `1`?

Reply:
+1